testing/test_process_result.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from manageiq_client.api import Collection, ActionContainer, Action
+
+
+def test_get_entity_from_href():
+    url_base = "http://example.com/api/"
+    test_data = [
```

Comment:
can use parametric test (https://docs.pytest.org/en/latest/parametrize.html) for clearer error attribution if one of these fails.